docs/solidity-by-example.rst

```diff
@@ -645,99 +645,71 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-In this section we will learn how to build a simple implementation
-of a payment channel. It use cryptographics signatures to make
+In this section we will learn how to build an example implementation
+of a payment channel. It use cryptographic signatures to make
 repeated transfers of Ether between the same parties secure, instantaneous, and
-without transaction fees. To do it we need to understand how to
+without transaction fees. For the example, we need to understand how to
 sign and verify signatures, and setup the payment channel.
 
 Creating and verifying signatures
 =================================
 
 Imagine Alice wants to send a quantity of Ether to Bob, i.e.
 Alice is the sender and the Bob is the recipient.
+.. TODO: What does the below mean?
```

Comment:
it means she does not have to send an actual transaction to the blockchain - is that what you mean?

Reply:
@chriseth I didn't write this, so it's not what 'I' mean, I was trying to understand what the original writer meant, 

> Alice only needs to send cryptographically signed messages off-chain
(e.g. via email) to Bob and it is similar to writing checks.

Just sounds kind of strange, but if it makes sense, then.